build/web_compose/Dockerfile

```diff
@@ -64,8 +64,8 @@ RUN pip3 install -r /workspace/server/requirements.txt
 COPY nl_app.py /workspace/nl_app.py
 COPY deploy/nl/. /datacommons/nl/
 COPY nl_server/requirements.txt /workspace/nl_server/requirements.txt
-RUN pip install --upgrade pip setuptools light-the-torch
-RUN ltt install torch --cpuonly
+RUN pip install --upgrade pip
+RUN pip3 install torch==2.2.1 --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
To check, this will increase the docker image (that was the original motivation for light-the-torch) how much?

Reply:
Nope, this also installs cpu only torch and the size is same.

Reply:
I should have found this way instead of light-the-torch :)